lib/restforce/collection.rb

```diff
@@ -31,6 +31,11 @@ def size
     end
     alias_method :length, :size
 
+    # Return whether the collection contains any objects.
```

Comment:
I think this will respond to `#any?` already and do the same thing, since it uses `Enumerable`. Can you check? Nice idea though.